commands/login/index.js

```diff
@@ -32,11 +32,30 @@ program
   .description('Log in and create your user configuration to use the other commands.')
   .action(async () => {
     let api;
-    try {
-      api = await retryP(signIn, isLockedError);
-    } catch (e) {
-      return logError(e);
+    const { email } = await inquirer.prompt([
+      { type: 'input', name: 'email', message: 'Enter Losant email:' }
+    ]);
+    api = await getApi();
+    const isSSoAccount = await api.auth.ssoDomain({ email });
+
+    if (isSSoAccount) {
+      const { token: apiToken } = await inquirer.prompt([
+        {
+          type: 'input',
+          name: 'token',
+          message: 'This email is attached to an SSO Account, please enter an API user token:'
```

Comment:
This is a run-on sentence.
```suggestion
          message: 'This email is attached to an SSO Account. Please enter an API user token:'
```

Reply:
inside of Losant we use the language
`Your account is linked to a Single Sign-On (SSO) provider.`

So I'd suggest something like this instead:

`This account is linked to a Single Sign-On (SSO) provider. Please create a CLI-Scoped User API Token in your Losant account, and then enter it here:`